config.js

```diff
@@ -0,0 +1,6 @@
+window.config = {
+    xKey: "",
+    xSoftwareName: "cardknox-sdk-js",
```

Comment:
The software name should have to be provided by the merchant, yo can pre-fill the SDK name and version